lib/inspec/json_profile_summary.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Inspec
+  #
+  # Inspec::JsonProfileSummary takes in certain information to identify a
+  # profile and then produces a JSON-formatted summary of that profile. It can
+  # return the results to STDOUT or a file. It is currently used in several
+  # places in the CLI such as `json`, `archive` and `artifact`.
+  #
+  #
+  module JsonProfileSummary
+    def self.produce_json(info:, dst: [], suppress_output: false)
```

Comment:
Could `dst` be documented or renamed? 
It looks like it can be an array or a filename; and the cases for empty array and a filename are implemented - what happens to a non-empty array?

Reply:
Hm, it looks like it should actually be an empty string default based on where it originally came from. Especially since being extracted it could be better documented.